### ISSUE TITLE
[SPIR-V] Run command in `CodeGenSPIRV/intrinsics.sm6_6.ishelperlane.hlsl` uses `ps_6_0` instead of `ps_6_6`

### ISSUE THREAD

**Issue by chaoticbob** [microsoft/DirectXShaderCompiler#6708]
Don't this matters in practice, but the run command at the top of this file is using `ps_6_0` instead of `ps_6_6`:
https://github.com/microsoft/DirectXShaderCompiler/blob/main/tools/clang/test/CodeGenSPIRV/intrinsics.sm6_6.ishelperlane.hlsl

```
// RUN: %dxc -T ps_6_0 -E main -fcgl  %s -spirv | FileCheck %s
```

Feature is SM 6.6 and file has sm6.6 in the name.

Just wanted to note this for the sake of consistency.

**Comment by llvm-beanz** [comment]
For DXIL we emulate `IsHelperLane` when targeting earlier shader models, so this feature is available in SM 6.0.

**Comment by chaoticbob** [comment]
Ahh, my bad. i thought it was strictly SM 6.6 and the SPIR-V backend was making it available to SM 6.0.
